project_task_code_portal/tests/test_portal.py

```
@@ -0,0 +1,279 @@
+                "res_model": "project.task",
+                "res_id": cls.task_1.id,
+                "partner_ids": [
+                    Command.link(cls.partner_portal.id),
+                ],
+            }
+        )
+        task_wizard.action_send_mail()
+
+        cls.host = "127.0.0.1"
+        cls.port = tools.config["http_port"]
+        cls.base_url = "http://%s:%d/my/tasks/" % (cls.host, cls.port)
```

Ruff (0.11.9)

Use format specifiers instead of percent format

(UP031)